tools/toolchains/arm.py

```diff
@@ -66,6 +66,9 @@ def __init__(self, target, notify=None, macros=None,
         self.asm = [main_cc] + self.flags['common'] + self.flags['asm']
         self.cc = [main_cc] + self.flags['common'] + self.flags['c']
         self.cppc = [main_cc] + self.flags['common'] + self.flags['c'] + self.flags['cxx']
+        if target.name in ["REALTEK_RTL8195AM"]:
```

Comment:
I dont think a target should change compilation flags. From ARMCC "The --enum_is_int option is not recommended for general use."

I guess without this flag the linker fails as not compatible enum sizes? If anyone can provide more information, and how we can target this one.